pkg/cloud/azure/azure.go

```diff
@@ -173,6 +173,10 @@ func CloudConfigTransformer(source string, infra *configv1.Infrastructure, netwo
 		cfg.VMType = azureconsts.VMTypeStandard
 	}
 
+	// Ensure we are checking each service node port when making health checks
+	// on service load balancers.
+	cfg.ClusterServiceLoadBalancerHealthProbeMode = azureconsts.ClusterServiceLoadBalancerHealthProbeModeServiceNodePort
```

Comment:
This should be the shared option, not the node port. Node port is the old broken behaviour

Reply:
Fixed.